motd/motd.yaml

```diff
@@ -146,6 +146,19 @@ oasis_branchoff6.21.23:
         action: DELAY(1)
         announcement: true
 
+provenance_feature:
+        package: "esm_tools"
+        versions: "<6.35"
+        message: "\x1b[1m\x1b[96mCONFIGURATION IS HISTORY!\x1b[0m\n
+
+            A useful \x1b[96mnew feature to show configuration history \x1b[0m is now\n
+            ready to be used! To get started, make sure you have at least version 6.35\n
+            and \x1b[96mmake sure to reinstall the package\x1b[0m since new depdency libraries\n
```

Comment:
```suggestion
            and \x1b[96mmake sure to reinstall the package (cd <PATH>/esm_tools; ./install.sh)\x1b[0m since new depdency libraries\n
```